packages/fxa-settings/src/components/Settings/AppLayout/index.tsx

```diff
@@ -30,6 +31,7 @@ export const AppLayout = ({ children }: AppLayoutProps) => {
         </main>
       </div>
       <Footer />
+      <div id="body-bottom" className="block mobileLandscape:hidden" />
```

Comment:
The Banner is very far down on the page for mobile, especially for 'settings' now that the banner is not sticky. I understand that's preferable for a11y - just making sure that we're okay with users potentially not seeing the banner (it will be below the initial screen view for most phones except `max` size devices).

![image](https://github.com/mozilla/fxa/assets/22231637/4c5e03d9-1dc5-41c7-a466-030762033cbe)

Reply:
I also raised this concern. I'll raise it again though.

Reply:
Thanks again for bringing this up. I think you are right it would largely go unnoticed on mobile, so for the time being I've gone ahead and fixed this by making the banner fixed in the mobile view, and thereby making it feel 'sticky'. I've tested on multiple device factors and this seems to work pretty well overall. There was a case (iPhone SE) where it obscured the screen, but I guess this is what's the close functionality is for.

Reply:
I agree the sticky footer looks better than the static footer. I am kind of curious if it's better than a sticky header though - I think either way FXA-7425 would be re-opened and maybe some others, but I'm not sure what the workaround would be to fix it in all cases and would just go with whatever UX is recommending.

Reply:
Just to be clear the header is still static, and the footer is now sticky.